pkg/lvm/lvm_util.go

```diff
@@ -518,8 +518,8 @@ func ReloadLVMMetadataCache() error {
 // groups in the node.
 //
 //If we will call this function from collector.go for metrics collection, exporter will be set as true otherwise false.
-func ListLVMVolumeGroup(exporter bool) ([]apis.VolumeGroup, error) {
-	if !exporter {
+func ListLVMVolumeGroup(reloadCache bool) ([]apis.VolumeGroup, error) {
+	if !reloadCache {
```

Comment:
If reloadCache is true, we need to reload the lvm metad cache. Can you invert the condition here and all the other places where the func is being used?